whois-commons/src/main/java/net/ripe/db/whois/common/ClockDateTimeProvider.java

```diff
@@ -13,18 +13,19 @@
 @Primary
 @Component
 public class ClockDateTimeProvider implements DateTimeProvider {
+
     @Override
     public LocalDate getCurrentDate() {
-        return LocalDate.now();
+        return getCurrentZonedDateTime().toLocalDate();
     }
 
     @Override
     public LocalDateTime getCurrentDateTime() {
-        return LocalDateTime.now();
+        return getCurrentZonedDateTime().toLocalDateTime();
     }
 
     @Override
-    public ZonedDateTime getCurrentDateTimeUtc() {
+    public ZonedDateTime getCurrentZonedDateTime() {
```

Comment:
There shouldn't be any difference between LocalDateTime and ZonedDateTime, is ZonedDateTime used for the 'Z' timezone?

Reply:
> is ZonedDateTime used for the 'Z' timezone?

Yes. It is the current time that includes the time zone related info

Reply:
OK (assuming we still need both)